circumstellar-habitable-zone-simulator/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "circumstellar-habitable-zone-simulator",
+  "version": "1.0.1",
+  "description": "Circumstellar Habitable Zone Simulator",
+  "private": true,
+  "main": "main.js",
+  "scripts": {
+    "test": "jest",
+    "serve": "webpack serve --mode development",
+    "dev": "webpack --mode development --watch",
+    "build": "webpack --mode production",
+    "eslint": "eslint src/*.js src/*.jsx"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ccnmtl/astro-simulations.git"
+  },
+  "keywords": [
+    "astronomy"
+  ],
+  "author": "Nik Nyby",
+  "license": "GPL-3.0+",
+  "bugs": {
+    "url": "https://github.com/ccnmtl/astro-simulations/issues"
+  },
+  "homepage": "https://github.com/ccnmtl/astro-simulations#readme",
+  "dependencies": {
+    "@babel/cli": "^7.5.5",
+    "eslint-plugin-security": "^1.4.0",
+    "pixi.js": "~5.3.0",
```

Comment:
I would use the latest version of pixi to start: 6.0.4. There's a small bug with SVG loading in 6.0.4, but it's easy to work around, and will be fixed soon anyways: https://github.com/pixijs/pixijs/pull/7478